app/views/content/subjects/physics/_article.html.erb

```diff
@@ -153,7 +153,7 @@
 </p>
 
 <p>
-  You could also be eligible for a scholarship of £29,000 or a bursary of £27,000.
+  You could also be eligible for a scholarship of £29,000 or a bursary of £27,000 if you start your teacher training between September 2023 and July 2024.
```

Comment:
I think this sounds a bit like if you start it in Sept 2024, you definitely won't get anything?

Last year we also added this - do we need something similar?
Financial support for courses starting in 2023 will be announced in autumn 2022.

(see https://github.com/DFE-Digital/get-into-teaching-app/pull/2837)